### ISSUE TITLE
Sensors cannot be found after pc startup from eco mode (bios setting)

### ISSUE THREAD

**Issue by ThimoDEV** [Rem0o/FanControl.Releases#1762]
**Describe the bug**
[A clear and concise description of what the bug is.  Screenshots are worth a thousand words. Hardware support requests do NOT count as a bug. See https://github.com/Rem0o/FanControl.Releases#issues-and-hardware-compatibility.]

I have my pc in power save mode/eco mode. Its the special bios setting that disables almost all hardware. After I then turn on my pc I get the below missing sensors error. A restart of Fancontrol doesn't solve it, only restarting my whole pc. The only fans I always see are my gpu fans.

![image](https://user-images.githubusercontent.com/68739517/236693995-942a9f66-bec4-404e-8645-1cdd6a07dc8c.png)

After I try Refresh sensor detection it says: initialization failed. If I then try it a second time it finishes but dooesn't add the fans.

**Is there a log.txt file next to FanControl.exe with recent date entries?**
No.

**Relevant hardware specs and setup**
MSI x570
AMD 5950x
RTX 3080

**Comment by Rem0o** [comment]
That eco-mode most likely turns some required features for FanControl to pick up the control chip.  I'm pretty sure you'll have to turn that off.

**Comment by ThimoDEV** [comment]
I believe a few versions back it didn't happen.  What do you think makes the eco mode restart different from a normal startup causing the sensor issue?

**Comment by Rem0o** [comment]
I have no idea what/how that eco mode affects regular boot up.   Clearly it messes with some stuff.  If you turn it off, does it works as expected?

**Comment by ThimoDEV** [comment]
Yes when my pc doesn't go into the eco sleep mode, fancontrol keeps working as expected. One thing I did notice was that if my pc went into sleep mode, I used it again (fancontrol sensors lost) and it went a second time into sleep mode, then the second time I started using my pc the sensors would be found again

**Comment by UnicycleUnicorn** [comment]
> Yes when my pc doesn't go into the eco sleep mode, fancontrol keeps working as expected. One thing I did notice was that if my pc went into sleep mode, I used it again (fancontrol sensors lost) and it went a second time into sleep mode, then the second time I started using my pc the sensors would be found again

Just tried this and putting my pc to sleep and waking it up again resolved the sensor issue... weird
